dtreeviz/models/spark_decision_tree.py

```diff
@@ -99,7 +100,7 @@ def get_features(self) -> np.ndarray:
         return self.features
 
     def criterion(self) -> str:
-        return self._get_tree_model_parameter_value("impurity")
+        return criterion_remapping.get(self._get_tree_model_parameter_value("impurity"), self._get_tree_model_parameter_value("impurity"))
```

Comment:
A small issue but might be better to simplify this into a function call. As above.